azurerm/internal/services/securitycenter/security_center_assessment_policy_resource_test.go

```diff
@@ -67,6 +67,21 @@ func TestAccSecurityCenterAssessmentPolicy_update(t *testing.T) {
 	})
 }
 
+func TestAccSecurityCenterAssessmentPolicy_categories(t *testing.T) {
```

Comment:
is there a reason we didn't just add this to the complete function (this its test being unset in the update)?

Reply:
Added it to complete function.